DOL.WHD.Section14c.Web/webpack.production.config.js

```diff
@@ -78,7 +84,8 @@ module.exports = {
   plugins: [
     new webpack.ProvidePlugin({
       $: 'jquery',
-      jQuery: 'jquery'
+      jQuery: 'jquery',
+      log: 'loglevel'
```

Comment:
Since `log` is being stripped out by `WebpackStripLoader`, does it still need to be injected by the `Provide` plugin?

Reply:
you are correct. I removed it in the production.config